### PR TITLE
DEMO: Add Custom Site to Sharing Overlay

### PR DESCRIPTION
**Pull request by waxidiotic**

### CODE REVIEW

**Comment by waxidiotic** [comment]
@josiekeller FYI

File: demos/developer-showcase/share-to-custom-site/css/style.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/* Add your CSS here
+Below are examples for how to customize the player skin
+*/
+
+
+/* Set the global Background color of the player elements */
+.jw-skin-seven .jw-background-color {
+	background: #0e4d8f !important;
```
Comment (joshuahatcher) [inline demos/developer-showcase/share-to-custom-site/css/style.css]: Avoid `!important` if you can.

File: demos/developer-showcase/share-to-custom-site/index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+You can set up a demo using our sample video and player or input your own.
+Ex: //content.jwplatform.com/players/{MediaID}-{PlayerID}.js
+
+MediaID and PlayerID are available in your Account Dashboard. -->
+
+<!-- <script src="//content.jwplatform.com/players/xJ7Wcodt-xDYBr4fw.js"></script> -->
+<script src="//content.jwplatform.com/libraries/GCSf7mm7.js"></script>
+
+<div id="player"></div>
```
Comment (joshuahatcher) [inline demos/developer-showcase/share-to-custom-site/index.html]: Generally good practice to put a newline at the end of files.

File: demos/developer-showcase/share-to-custom-site/js/main.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  }
+});
+
+function shareToWhatsApp() {
+  // If a permalink is configured for a video in the dashboard, it will share it.
+  // If not, fall back to sharing the page's URL (or preview page if using a feed).
+  var linkToShare = playerInstance.getPlaylistItem().link || window.location;
+
+  window.location.href = 'https://api.whatsapp.com/send?text=' + linkToShare;
+}
```
Comment (joshuahatcher) [inline demos/developer-showcase/share-to-custom-site/js/main.js]: Same here w/r/t newline

File: demos/developer-showcase/share-to-custom-site/css/style.css (hunk 1 above)
Comment (waxidiotic) [reply demos/developer-showcase/share-to-custom-site/css/style.css]: @joshuahatcher That's from the demo-starter-kit.

https://github.com/jwplayer/jwdeveloper-demos/blob/developer-showcase/demos/developer-showcase/demo-starter-kit/css/style.css

File: demos/developer-showcase/share-to-custom-site/index.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+-->
+
+<!-- This demo uses a single line embed. It contains a MediaID and PlayerID.
+
+You can set up a demo using our sample video and player or input your own.
+Ex: //content.jwplatform.com/players/{MediaID}-{PlayerID}.js
+
+MediaID and PlayerID are available in your Account Dashboard. -->
+
+<!-- <script src="//content.jwplatform.com/players/xJ7Wcodt-xDYBr4fw.js"></script> -->
```
Comment (joshuahatcher) [inline demos/developer-showcase/share-to-custom-site/index.html]: Delete commented-out code

**Comment by waxidiotic** [comment]
@monibons any way you can take a look at this? it's gathering dust lol

**Pull request closed without merging**